lib/Doctrine/Search/Query.php

```diff
@@ -36,7 +36,7 @@ class Doctrine_Search_Query
     /**
      * @var Doctrine_Table $_table          the index table
      */
-    protected $_table = array();
+    protected $_table;
```

Comment:
Not sure why this was ever set to an empty array, as it's never accessed as such and only stores one Doctrine_Table object.